experiments/snippets/smartAdopt/README.md

```diff
@@ -0,0 +1,19 @@
+# Smart Adopt
+Smart adopt is a library capable of intelligently adopting records and fields between different models. There are many different use cases for implementing this library, from mass-creating junction records to simplifying a 'multi-select' experience. It's capable of a number of things including updating existing records which match between 2 models, create new records in the destination model where there are unmatched records, or mark records for deletion in the destination model where none exists in the source. It can also specify key fields arbitrarily between the 2 models and selectively choose which data fields are or are not included in a given upsert operation.
+
+## Installation
+Simply copy the contents of SmartAdopt.js into a new snippet titled "SmartAdopt", and copy the contents of SmartAdopt.xml into the actionsequences node in your page XML. The purpose of the action XML is simply to provide declarative input parameters for our snippet.
```

Comment:
Hrm, I get the necessity of these kind of installation instructions, but it's unfortunate. Especially since Skuid by default uses self-closing tags for its empty page resources. We might need more detail here similiar to this, or at least some kind of note:

```

Replace `<actionsequences/>` with `<actionsequences> <snippet code here> </actionsequences>`
```

Having an actual sample _page_ would also be cool, even if it were blank. (Dream scenario: a sample page illustrating one of our use cases, but I recognize that's scope creep)